### DISCUSSION TITLE
How to send and receive data with "Link" tag in React?

### DISCUSSION THREAD

**Discussion by amori95917** [community/community#49792] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have sent data with <Link> tag.
`<Link to = "/community" state = {data}/>`

But how to receive data in corresponded component?

**Comment by 03timothyash** [comment]
Gracias por tu atención.
YO lo compro.

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by 03timothyash** [comment]
Es muy importante para solucionar este problema de tipo almacenamiento.

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________
From: Alexander Mori ***@***.***>
Sent: Saturday, March 11, 2023 7:39:32 AM
To: community/community ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [GitHub Community] How to send and receive data with "Link" tag in React? (Discussion #49792)


Select Topic Area

Question

Body

I have sent data with tag.
<Link to = "/community" state = {data}/>

But how to receive data in corresponded component?

**Comment by 03timothyash** [comment]
Que tengas un buen día.

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________
